apps/src/util/thumbnail.js

```diff
@@ -20,33 +17,47 @@ const THUMBNAIL_SIZE = 180;
 let lastCaptureTimeMs = 0;
 
 /**
- * Converts the contents of an SVG element into an image, shrinks it to
- * width and height equal to THUMBNAIL_SIZE, and saves it to
- * @param {SVGElement} svg Svg element to capture the contents of.
- * @param {number} minCaptureIntervalMs Minimum allowable time between
- *   thumbnail image captures in milliseconds.
+ * Returns true if this level is a project level owned by this user, it is not a
+ * shared or embedded level, and enough time has passed since the last capture.
+ * @param {number} minCaptureIntervalMs Only capture if it has been at least
+ *   this many milliseconds since the last capture. Optional.
+ * @returns {boolean}
  */
-export function captureThumbnailFromSvg(svg, minCaptureIntervalMs) {
+ function shouldCapture(minCaptureIntervalMs) {
   const {isShareView, isEmbedView} = getStore().getState().pageConstants;
   if (!project.getCurrentId() || !project.isOwner || isShareView || isEmbedView) {
-    return;
+    return false;
   }
 
   // Skip capturing a screenshot if we just captured one recently.
-  if (Date.now() - lastCaptureTimeMs < minCaptureIntervalMs) {
+  const intervalMs = Date.now() - lastCaptureTimeMs;
+  if (minCaptureIntervalMs > 0 && intervalMs < minCaptureIntervalMs) {
```

Comment:
Why do we have the `minCaptureIntervalMs > 0` check here? Is it just to protect against the case where `minCaptureIntervalMs` is undefined? We could also provide a default value for this param, i.e.
`function shouldCapture(minCaptureIntervalMs=0) {`